src/agent/prompter.js

```diff
@@ -1,12 +1,14 @@
 import { readFileSync, mkdirSync, writeFileSync} from 'fs';
-import { Gemini } from '../models/gemini.js';
-import { GPT } from '../models/gpt.js';
 import { Examples } from '../utils/examples.js';
 import { getCommandDocs } from './commands/index.js';
 import { getSkillDocs } from './library/index.js';
 import { stringifyTurns } from '../utils/text.js';
 import { getCommand } from './commands/index.js';
 
+import { Gemini } from '../models/gemini.js';
+import { GPT } from '../models/gpt.js';
+import { Claude } from '../models/claude.js';
```

Comment:
Can we make these imports conditional on the `model_name` so that you could still run the bot without all the packages installed?

Reply:
I don't think there is an easy way to do this with node. I don't think the packages are too much to install, and they don't require an api key to install. It'd probably be more work than its worth to install them conditionally